ex35.py

```diff
@@ -63,49 +65,49 @@ def cthulu_room():
         cthulu_room()
 
 
-tries = 0 # how to get this into the function and used in the nested functions?
-          # maybe not possible? blech
 def cthulus_mind():
+    tries = 0
+
+    while tries < 11:
+        if tries == 0:
+            print "Your vision begins to blur and goes dark."
+            print "You hear in the distance a familiar bellow:"
+            print "Mortal... completing this task will prove your sanity in the face of a god..."
+            print "ENTERRRRRRRRRRRR"
+            cthulu_mind_test()
+            tries += 1
+        elif 0 < tries < 10:
+            print "You may try again. Beware, these are not unlimited..."
+            print 10 - tries
+            cthulu_mind_test()
+            tries += 1
+        else:
+            print "You go insane and begin to flee!"
+            start()
+    return

```

Comment:
The "return" here is just to make the end of the while loop clear.  In a language with braces instead of whitespace, the end of the while loop would be explicit.